missilewars-plugin/src/main/java/de/butzlabben/missilewars/Config.java

```diff
@@ -148,25 +158,19 @@ public static void load() {
         return ret;
     }
 
+    /**
+     * This methode get the material of the block to start the missile.
```

Comment:
Change to something like this: "This methods gets the minecraft material type of the block to start missiles"